app/models/highlight.rb

```diff
@@ -29,4 +37,10 @@ def normalize_source_ids
       source_parent_ids.map!(&:downcase)
```

Comment:
do we need to trim/strip the IDs too?

Reply:
Also where do we document that for openstax_page the source IDs have such and such form?  Do we need to check for that form?  E.g. is `@12` version stuff included?

Reply:
need to ask about this...i had wondered this myself.

Reply:
added a custom validation for source_parent_ids to the Highlight model.